docs/reference/search/search-your-data/retrievers-examples.asciidoc

```diff
@@ -198,23 +223,875 @@ GET /retrievers_example/_search
        "field": "year",
        "inner_hits": {
            "name": "topic related documents",
-           "_source": ["text", "year"]
+           "_source": ["year"]
        }
    },
-   "_source": ["text", "topic"]
+    "_source": false
+}
+----
+// TEST[continued]
+
+Which would return the following response collapsed results
+
```

Comment:
```suggestion
.Example response
[%collapsible]
==============
```

consider putting these responses inside collapsibles (nothing to do with "collapsed results 😛 )

Reply:
yeap makes sense! will wrap all responses into collapsibles!